README.md

```diff
@@ -134,14 +134,14 @@ response instead.
 1. Create a `main.py` file with the following contents:
 
    ```python
-   def hello(request):
-        return "Hello world!"
+   def hello(event, context):
```

Comment:
I would stick to `data, context` to update this Pub/Sub quickstart. See other comments for more details.

Reply:
+1